pkg/util/util_test.go

```diff
@@ -20,6 +19,10 @@ var (
 	TestArchivePath string
 	// TestArchiveExtractDir is the directory in the archive to extract
 	TestArchiveExtractDir = "wordpress-0.4.0/"
+
+	// The image here can be any image, it just has to exist so it can be used for labels, etc.
+	TestRouterImage = "drud/nginx-proxy"
+	TestRouterTag   = "test-do-not-delete"
```

Comment:
Seems like https://hub.docker.com/_/busybox/ is perfect for this sort of use case. If I read the code correctly we're only using it to test that we can read env vars?